### ISSUE TITLE
Themes app doesn't starting

### ISSUE THREAD

**Issue by rapedrix** [Euphoria-OS/Bug-tracker#46]
![screenshot_2015-09-13-02-09-46](https://cloud.githubusercontent.com/assets/14254806/9834516/d26a0f98-59bc-11e5-88df-ab91932766ee.png)
Every time I try to open the app, it crashes, im in Moto G 2013 using EuphoriaOS 5.1.1 20150904

**Comment by dsmitty166** [comment]
Have you done a clean flash? If not I will be closing this, this is probably a one off

**Comment by rapedrix** [comment]
Yes, i did the full wipe (system, data, cache and dalvik cache) and i flashed the rom and gapps like 2 times and the theme engine didnt worked .
